docs/posts/wow-analysis/index.qmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "Analsis of World of Warcraft Data"
```

Comment:
```suggestion
title: "Analysis of World of Warcraft Data"
```

Reply:
This is addressed in the new commit, thank you! Can't believe I missed that one.